### DISCUSSION TITLE
For invariant tests, fail_on_revert = false, but only run ONE runs!!

### DISCUSSION THREAD

**Discussion by CH-coding-fire** [Cyfrin/foundry-full-course-cu#1961] [Q&A]
I run invariant test, I expect it to run 64 times, because that's what I set in foundry.tomo.
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/67637342/a3459d29-e006-4b9c-8f84-cd81ed8e5d1a)

But in my test it only run ONCE.

![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/67637342/15567e76-0486-4f01-96aa-065beba5dd0e)
this is frustrating, help!!

**Comment by EngrPips** [comment]
Hello @CH-coding-fire, as a first step, please run `foundryup` and retry running the test again. Foundry sometimes behaves abnormally when it needs to be updated.

**Reply by CH-coding-fire** [reply]
Hi, I have tried this, but it does not change anything

**Reply by EngrPips** [reply]
That's weird, actually. I have never run into this error before. I will suggest you make an issue on the `Foundry` official Github `https://github.com/foundry-rs/foundry`
